### DISCUSSION TITLE
blog: use description meta as excerpt

### DISCUSSION THREAD

**Discussion by StefanoCecere** [squidfunk/mkdocs-material#7070] [Ideas]
if the post description meta is set, it used used instead of the full post / excerpt method

it could be a third option of 

```
plugins:
  - blog:
      post_excerpt: description
```

**Comment by squidfunk** [comment]
Thanks for suggesting! Unfortunately, this is not a good idea for us to support, as it is only allowed to use plain text (not Markdown) inside the description, because it is also used for the meta `description` tag, as well as for social cards. No HTML, no Markdown, as it just cannot be supported. However, this can either be implemented with hooks or theme extension, i.e., custom overrides ☺️
